### PR TITLE
[Python 2->3] Run futurize as part of butler.py lint

### PR DESCRIPTION
**Pull request by mbarbella-chromium**
PTAL when you have a chance.

### CODE REVIEW

File: src/local/butler/lint.py [hunk 1]
```diff
@@ -185,6 +185,14 @@ def execute(_):
   for file_path in py_changed_file_paths:
     _execute_command_and_track_error('pylint ' + file_path)
     _execute_command_and_track_error('yapf -d ' + file_path)
+    futurize_output = _execute_command_and_track_error(
+        'futurize -0 -x libfuturize.fixes.fix_unicode_keep_u '
```
Comment (inferno-chromium) [inline src/local/butler/lint.py]: please put the fixes in a list var global / butler constants.py and then just add them here with -x

File: src/python/platforms/fuchsia/util/device.py [hunk 2]
```diff
@@ -12,8 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Utilities for managing Fuchsia devices."""
+from __future__ import absolute_import
+from builtins import object
+from builtins import range
 
-from host import Host
+from .host import Host
```
Comment (inferno-chromium) [inline src/python/platforms/fuchsia/util/device.py]: can you move it after the system imports.

File: src/local/butler/lint.py [hunk 3]
```diff
@@ -185,6 +185,14 @@ def execute(_):
...
+        '-x libfuturize.fixes.fix_future_builtins -x lib2to3.fixes.fix_dict '
+        '-x lib2to3.fixes.fix_ws_comma -x lib2to3.fixes.fix_idioms ' +
```
Comment (inferno-chromium) [inline src/local/butler/lint.py]: Why fix_ws_comma ?

Comment (mbarbella-chromium) [reply src/local/butler/lint.py]: idioms and ws_comma are disabled by default and not explicitly disabling them makes the tool print an extra log line for each. Just did it to save some spam here. I'll leave a comment explaining that when addressing the first comment.

File: src/local/butler/lint.py (hunk 1 above)
Comment (mbarbella-chromium) [reply src/local/butler/lint.py]: Done

File: src/python/platforms/fuchsia/util/device.py (hunk 2 above)
Comment (mbarbella-chromium) [reply src/python/platforms/fuchsia/util/device.py]: Done

**Pull request merged**